lib/ansible/modules/web_infrastructure/jenkins_job.py

```diff
@@ -325,12 +329,10 @@ def get_result(self):
 
 def test_dependencies(module):
     if not python_jenkins_installed:
-        module.fail_json(msg="python-jenkins required for this module. "
-                         "see http://python-jenkins.readthedocs.io/en/latest/install.html")
```

Comment:
Is the URL needed? It doesn't provide a ton of information, so I could see why it can be safely omitted.

Reply:
I'll keep it in there, has some info on other package managers that may be useful.